src/app/pages/apps/asset-profile/asset-profile-list/asset-profile-list.component.ts

```diff
@@ -0,0 +1,139 @@
+import { AfterViewInit, Component, Input, OnDestroy, OnInit, ViewChild } from '@angular/core';
+import { Observable,ReplaySubject, Subject, finalize, takeUntil } from 'rxjs';
+import { filter} from 'rxjs/operators';
+import { MatTableDataSource } from '@angular/material/table';
+import { MatPaginator } from '@angular/material/paginator';
+import { MatSort } from '@angular/material/sort';
+import { TableColumn } from '../../../../../@vex/interfaces/table-column.interface';
+import { aioTableLabels } from '../../../../../static-data/aio-table-data';
+import { SelectionModel } from '@angular/cdk/collections';
+import { fadeInUp400ms } from '../../../../../@vex/animations/fade-in-up.animation';
+import { MAT_FORM_FIELD_DEFAULT_OPTIONS, MatFormFieldDefaultOptions } from '@angular/material/form-field';
+import { stagger40ms } from '../../../../../@vex/animations/stagger.animation';
+import { UntypedFormControl } from '@angular/forms';
+import { UntilDestroy, untilDestroyed } from '@ngneat/until-destroy';
+import { PortalService } from 'src/app/core/services/portal.service';
+import { Router } from '@angular/router';
+import { IAssetProfile } from 'src/app/core/models/asset-profile';
+
+@UntilDestroy()
+@Component({
+  selector: 'vex-aio-table',
```

Comment:
Make sure the if you do copy and paste the component. You also change the name of it.

Reply:
Noted on this sir, Applied.